workflows/prognostic_c48_run/runtime/config.py

```diff
@@ -13,6 +13,7 @@
 from runtime.steppers.nudging import NudgingConfig
 from runtime.steppers.machine_learning import MachineLearningConfig
 from runtime.steppers.prescriber import PrescriberConfig
+from runtime.emulator import Config as OnlineEmulatorConfig
```

Comment:
Is it unclear? I needed to rename the `Config` name to something, or I could do something like
```
import runtime.emulator
```
and then refer to `runtime.emulator.Config`.


BTW, we do rename things like this other places e.g.:
```
import fv3viz as viz
import numpy as np
import diagnostic_utils as utils
```

Reply:
Note this is all my attempt to avoid redundant names like `runtime.emulator.online.submodule.EmulatorOnlineSubmoduleConfig`

Reply:
yeah, i understand the goal of reducing redundancy. I just found it more difficult to distinguish between the `fv3fit.emulation.thermobasis.emulator.Config` and the `runtime.emulator.Config` due to them having the same name when they are defined. I think the examples you show are a bit different, since this is the renaming of packages. I also think that being consistent with the current pattern within `runtime/config.py` is a reasonable default behavior.

Reply:
I think one could make arguments either way...my way is suggested by books like Clean Code and online examples I have seen. Otherwise I'm not sure it's worth debating `runtime.emulator.Config` vs `runtime.emulator.RuntimeEmulatorConfig`. I thing the former is an elegant solution with exactly the same semantic meaning. If you are in a file `runtime/emulator.py` then I think it should be clear that the object named `Config` is used locally.

I will remove the renaming.

Reply:
See 5755a1b8

Reply:
FWIW I do think the fv3fit subpackage naming could be improved...it's too deeply nested, and the code in `fv3fit.emulation` is pretty general purposes to any prediction task involving thermodynamic variables.